gmail/Package.md

```diff
@@ -1,309 +1,21 @@
-Connects to Gmail from Ballerina.
+Connects to [Gmail](https://www.google.com/gmail/about/) from Ballerina.
 
-## Module Overview
+## Package overview
 
-Ballerina Gmail Connector provides the capability to send, read and delete emails through the Gmail REST API. It also provides the ability to read, trash, untrash and delete threads, ability to get the Gmail profile and mailbox history, etc. The connector handles OAuth 2.0 authentication.
+The `ballerinax/googleapis.gmail` is a [Ballerina](https://ballerina.io/) connector for Gmail.
```

Comment:
Should we use the connector name?

Reply:
This is the common format in the guideline doc. https://docs.google.com/document/d/1WZg1ix9MpKDTQ_OOm3NVqhQ5L0VGNaUQ1LRmjp-SMO4/edit#heading=h.uoc8qk37qag0.